pkgs/desktops/gnome-3/extensions/taskwhisperer/default.nix

```diff
@@ -37,6 +37,7 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "GNOME Shell TaskWarrior GUI";
     license = licenses.gpl3Plus;
+    broken = true; # Likely broken https://github.com/cinatic/taskwhisperer/issues/60
```

Comment:
I think the issue is about trying to use the extension version for GNOME 3.32 in GNOME 3.30. v12 claims to support 3.32: https://github.com/cinatic/taskwhisperer/blob/v12/taskwhisperer-extension@infinicode.de/metadata.json

Reply:
@worldofpeace was trigger mark as broken happy :+1:

Reply:
It would also allow us to switch `installPhase` for `makeFlags = ["INSTALLBASE=${placeholder ''out''}/share/gnome-shell/extensions"]`.

Reply:
Nice, I'll resync the patch also.